### PR TITLE
test: initial layout management tests

### PR DESCRIPTION
**Pull request by MaximKhlobystov**
### What does this PR do?

This PR adds a few tests for layout management.

In order to avoid false failures due to the video dock, the videos are masked with pink boxes:
masked

### CODE REVIEW

File: bigbluebutton-tests/playwright/core/elements.js [hunk 1]
```diff
@@ -428,6 +428,13 @@ exports.simpleModal = 'div[id="simpleModal"]';
 exports.sharedNotesBackground = 'div[data-test="notes"]';
 exports.whiteboardOptionsButton = 'button[data-test="whiteboardOptionsButton"]';
 
+// Layout management
+exports.layoutSettingsModalButton = 'li[data=test="layoutSettingsModalButton"]';
+exports.focusOnPresentation = 'button[data="presentationFocusLayout"]';
+exports.focusOnVideo = 'button[data="videoFocusLayout"]';
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/core/elements.js:434]:
```suggestion
exports.layoutSettingsModalButton = 'li[data-test="layoutSettingsModalButton"]';
exports.focusOnPresentation = 'button[data-test="presentationFocusLayout"]';
exports.focusOnVideo = 'button[data-test="videoFocusLayout"]';
```

File: bigbluebutton-tests/playwright/layouts/focusOnPresentation.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const { expect } = require('@playwright/test');
+const Page = require('../core/page');
+const e = require('../core/elements');
+const { ELEMENT_WAIT_LONGER_TIME } = require('../core/constants');
+const { MultiUsers } = require('../user/multiusers');
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/layouts/focusOnPresentation.js:5]: imports not being used
```suggestion
const { expect } = require('@playwright/test');
const e = require('../core/elements');
const { MultiUsers } = require('../user/multiusers');
```

File: bigbluebutton-tests/playwright/layouts/focusOnVideo.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const { expect } = require('@playwright/test');
+const Page = require('../core/page');
+const e = require('../core/elements');
+const { ELEMENT_WAIT_LONGER_TIME } = require('../core/constants');
+const { MultiUsers } = require('../user/multiusers');
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/layouts/focusOnVideo.js:5]: imports not being used
```suggestion
const { expect } = require('@playwright/test');
const e = require('../core/elements');
const { MultiUsers } = require('../user/multiusers');
```

File: bigbluebutton-tests/playwright/layouts/focusOnVideo.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    const modPageCameraDockLocator = await this.modPage.getLocator(e.cameraDock);
+    await expect(this.modPage.page).toHaveScreenshot('moderator1-focus-on-video.png', {
+      maxDiffPixels: 1000,
+      mask: [modPageCameraDockLocator],
+    });
+
+    const modPage2CameraDockLocator = await this.modPage2.getLocator(e.cameraDock);
+    await expect(this.modPage2.page).toHaveScreenshot('moderator2-focus-on-video.png', {
+      maxDiffPixels: 1000,
```
Comment (antonbsa) [inline bigbluebutton-tests/playwright/layouts/focusOnVideo.js:28]: with maxDiffPixels set to `1000` occasionally I got an error, but it's just the icons' color and meeting name:
image

I'd say to use `maxDiffPixelRatio: 0.005` instead in all screenshots comparisons (increasing the pixels diff a little bit will work, but I still prefer pixelRatio)

File: bigbluebutton-tests/playwright/layouts/layouts.spec.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+const { test } = require('@playwright/test');
```
Comment (daminebenq) [inline bigbluebutton-tests/playwright/layouts/layouts.spec.js:1]:
```
const { test } = require('@playwright/test');
const { FocusOnPresentation } = require('./focusOnPresentation');
const { FocusOnVideo } = require('./focusOnVideo');
const { MultiUsers } = require('../user/multiusers');
const { encodeCustomParams } = require('../customparameters/util');

const CUSTOM_MEETING_ID = 'layout_management_meeting';
const CUSTOM_STYLE = `userdata-bbb_custom_style=.presentationUploaderToast{display: none;}.currentPresentationToast{display:none;}`;

// Initialize mod page with provided params
const initModPage = async (page, context, options) => {
  const focus = new FocusOnPresentation(browser, context);
  await focus.initModPage(page, true, { customMeetingId: CUSTOM_MEETING_ID, customParameter: encodeCustomParams(CUSTOM_STYLE) });
  await focus.initModPage2(true, context, { customParameter: encodeCustomParams(CUSTOM_STYLE) });
};

test.describe.parallel('Layout management', () => {
  test('Focus on presentation', async ({ browser, context, page }) => {
    await initModPage(page, context);
    const focusOnPresentation = new FocusOnPresentation(browser, context);
    await focusOnPresentation.test();
  });
  
  test('Focus on video', async ({ browser, context, page }) => {
    const focusOnVideo = new FocusOnVideo(browser, context);
    await initModPage(page, context);
    await focusOnVideo.test();
  });
});
```

I thought a bit differently. What do you think @MaximKhlobystov ?

**Review by daminebenq — CHANGES_REQUESTED** [review]

File: bigbluebutton-tests/playwright/layouts/layouts.spec.js (hunk 5 above)
Comment (MaximKhlobystov) [reply bigbluebutton-tests/playwright/layouts/layouts.spec.js:1]: I don't think we should create multiple instances of FocusOnPresentation. But the idea of reusing the constants is good. Thanks for the feedback!

**Review by antonbsa — APPROVED** [review]
Sometimes I'm still facing failures due to the screenshots comparison (when only the avatar icon color and meeting name are different). I'd say we improve the ratio accuracy in an upcoming PR and merge this one!

**Pull request merged**